apisix/cli/config.lua

```diff
@@ -176,6 +176,8 @@ local _M = {
         ["cas-auth"] = "10m",
         ["ocsp-stapling"] = "10m",
         ["mcp-session"] = "10m",
+        status_report_standalone = "1m",
+        status_report = "1m"
```

Comment:
Is this necessary to use two different shared dicts? Since they won't be enabled at the same time and are (rightfully) functionally identical, this doesn't seem necessary.

Reply:
And why alone it is using `_` instead of `-`.